### DISCUSSION TITLE
UI: menu dropdown doesn't honor visible area and scroll position

### DISCUSSION THREAD

**Discussion by glintik** [grafana/grafana#41126] [General]
**What happened**:
When dashboard is long enough, dropdown menu for Panel Title doesn't honor scroll position and currently visible area:
<img width="717" alt="Screenshot 2021-10-29 at 13 12 31" src="https://user-images.githubusercontent.com/2947475/139418546-d47c81d7-7576-475c-8477-1e52640ac8b5.png">

**What you expected to happen**:
Dropdown menu should be opened above clicked element:
<img width="528" alt="Screenshot 2021-10-29 at 13 15 46" src="https://user-images.githubusercontent.com/2947475/139418731-e2279d1b-437e-4e09-9df5-8fa6aba55917.png">

**How to reproduce it (as minimally and precisely as possible)**:
Create long dashboard so scroll area shown.

**Environment**:
- Grafana version: v8.3.0-pre (fbdeffd340)
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser: macOS 10.14.6 (18G6042) & Firefox 93.0
- Grafana plugins:
- Others:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
